bin/GameIndex.dbf

```diff
@@ -1203,7 +1203,7 @@ Name   = Drakan - The Ancients' Gates
 Region = PAL-M5
 Compat = 5
 EETimingHack = 1 // Fixes flickering textures.
-[patches]
+[patches = 04F9D87F]
```

Comment:
This will cause the patch not to work with different crc ids using the same serial.